crates/ruff_python_ast/src/nodes.rs

```diff
@@ -2578,13 +2577,36 @@ impl Ranged for Identifier {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum Int {
+    /// A "small" number that can be represented as an `isize`. This exists as an optimization to
+    /// avoid allocating for the common case.
+    ///
+    /// `isize` matches the limitations of Python's `sys.maxsize`, in that it's 2^63 - 1 on 64-bit
+    /// systems, and 2^31 - 1 on 32-bit systems. (Python allows you to define integers larger than
+    /// `sys.maxsize`, but it seems convenient to choose a limit here that has some meaning in
+    /// Python.)
+    Small(isize),
```

Comment:
The choice of `isize` is semi-arbitrary, but the reasoning is laid out above. We could even use `i64`, it shouldn't matter much.

Reply:
I think we should use `i64` here and not `isize`. The size types are intended for pointer-like numbers: The length of an array or string. This use case is different because we don't want to represent a length but an absolute number. 

Not having to think about whether its an `i64` or `i32` depending on the target platform will make our life easier.